en/docs/develop/hosting-authentication-endpoint-on-a-different-server.md

```diff
@@ -158,7 +186,7 @@ Server).
     retry= "https://localhost:8443/authenticationendpoint/retry.do"
     ```
 
-6.  Import the public certificate of the identity server to the javaca certs (or web-serverstruststore) of the JVM that the authenticationendpoint is running.
+13. Import the public certificate of the identity server to the javaca certs (or web-serverstruststore) of the JVM that the authenticationendpoint is running.
```

Comment:
```suggestion
13. Import the public certificate of the identity server to the `javaca` certs (or `web-serverstruststore`) of the JVM where the authentication endpoint is running.
```

Reply:
What is `javaca` certs? Is this supposed to be a truststore?

Reply:
javaca is where Java stores public certificates of root CAs. It's not something related to IS. It's by default available in installed java